src/ucs/config/global_opts.c

```diff
@@ -191,6 +191,41 @@ static ucs_config_field_t ucs_global_opts_table[] = {
   "The list order decides the priority of the providers.",
   ucs_offsetof(ucs_global_opts_t, topo_prio), UCS_CONFIG_TYPE_STRING_ARRAY},
 
+ {"DISTANCE_LAT", "pxb:300ns,phb:300ns,sys:500ns",
+  "Estimated latency between system devices", 0,
+  UCS_CONFIG_TYPE_KEY_VALUE(UCS_CONFIG_TYPE_TIME,
+      {"pxb", "latency between devices with the same PCI domain and bus number",
+       ucs_offsetof(ucs_global_opts_t, dist_pxb_lat)},
+      {"phb",
+       "latency between devices with different PCI domains and/or bus numbers "
+       "residing on the same NUMA node",
+       ucs_offsetof(ucs_global_opts_t, dist_phb.latency)},
+      {"sys", "latency between devices residing on different NUMA nodes",
+       ucs_offsetof(ucs_global_opts_t, dist_sys.latency)},
+      {NULL}
+ )},
+
+ {"DISTANCE_BW", "pxb_k:19200MBs,pxb_max:3500MBs,phb:17000MBs,sys:220MBs",
+  "Estimated bandwidth between system devices", 0,
+  UCS_CONFIG_TYPE_KEY_VALUE(UCS_CONFIG_TYPE_BW,
+      {"pxb_k",
+       "coefficient of proportionality between the estimated bandwidth and the "
+       "distance between devices with the same PCI domain and bus number. The "
+       "bandwidth is inversely proportional to the distance.",
+       ucs_offsetof(ucs_global_opts_t, dist_pxb_bw_k)},
+      {"pxb_max",
+       "maximum estimated bandwidth between devices with the same PCI domain "
+       "and bus number",
+       ucs_offsetof(ucs_global_opts_t, dist_pxb_bw_max)},
+      {"phb",
+       "bandwidth between devices with different PCI domains and/or bus "
+       "numbers residing on the same NUMA node",
+       ucs_offsetof(ucs_global_opts_t, dist_phb.bandwidth)},
+      {"sys", "bandwidth between devices residing on different NUMA nodes",
+       ucs_offsetof(ucs_global_opts_t, dist_sys.bandwidth)},
+      {NULL}
```

Comment:
Maybe it worth using same with `nvidia-smi` distances' descriptions:

![image](https://github.com/openucx/ucx/assets/22097249/520e5e46-3162-45e2-a84e-ee35b5508688)